sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java

```diff
@@ -18,29 +18,265 @@
+          fillDateOfBirth(caze.getDateOfBirth(), Locale.ENGLISH);
+          selectSex("Male");
+          selectPresentConditionOfPerson(caze.getPresentConditionOfPerson());
+          fillDateOfSymptomOnset(caze.getDateOfSymptomOnset(), Locale.ENGLISH);
+          fillPrimaryPhoneNumber(caze.getPrimaryPhoneNumber());
+          fillPrimaryEmailAddress(caze.getPrimaryEmailAddress());
+          fillDateOfReport(date, Locale.ENGLISH);
+          fillPlaceDescription(caze.getPlaceDescription());
+        });
+
+    When(
+        "I fill second new case with for one person with specified date",
+        () -> {
+          LocalDate date = LocalDate.now();
+          caze = caseService.buildGeneratedCaseForOnePerson(firstName, lastName, dateOfBirth);
+          selectCaseOrigin(caze.getCaseOrigin());
+          fillExternalId(randomUUID_second_user.toString());
+          fillDisease(caze.getDisease());
+          selectResponsibleRegion(caze.getResponsibleRegion());
+          selectResponsibleDistrict(caze.getResponsibleDistrict());
+          selectResponsibleCommunity(caze.getResponsibleCommunity());
+          selectPlaceOfStay(caze.getPlaceOfStay());
+          fillFirstName(caze.getFirstName());
+          fillLastName(caze.getLastName());
+          fillDateOfBirth(caze.getDateOfBirth(), Locale.ENGLISH);
+          selectSex("Male");
+          selectPresentConditionOfPerson(caze.getPresentConditionOfPerson());
+          fillDateOfSymptomOnset(caze.getDateOfSymptomOnset(), Locale.ENGLISH);
+          fillPrimaryPhoneNumber(caze.getPrimaryPhoneNumber());
+          fillPrimaryEmailAddress(caze.getPrimaryEmailAddress());
+          fillDateOfReport(date, Locale.ENGLISH);
+          fillPlaceDescription(caze.getPlaceDescription());
+        });
+
+    When(
+        "I click on save case button and confirm Pick person in Case",
+        () -> {
+          String expectedTitle = "Pick or create person";
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(20);
+          String checkPopupTitle =
+              webDriverHelpers.getTextFromWebElement(PICK_OR_CREATE_PERSON_TITLE);
+          softly.assertEquals(
+              checkPopupTitle, expectedTitle, "Wrong popup title for Pick or create person");
+          softly.assertAll();
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_POPUP_CONTENT);
+        });
+
+    When(
+        "I filter Cases by created person name",
+        () -> {
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.fillInWebElement(
+              PERSON_ID_NAME_CONTACT_INFORMATION_LIKE_INPUT, firstName + " " + lastName);
+          webDriverHelpers.clickOnWebElementBySelector(CASE_APPLY_FILTERS_BUTTON);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
+        });
+
+    When(
+        "I select ([^\"]*) created case for person from Cases list",
```

Comment:
this will be flaky, because if another test runs in parallel, the list won't be the same

Reply:
i am running 3 tests in the same time which are using this method, everything works correctly